### PR TITLE
chore: mdadm, pkgs: update packages and replace eudev by systemd-udevd

### PR DESCRIPTION
**Pull request by dsseng**
Fix builds with new Talos packages versions

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -48,7 +48,7 @@ COMMON_ARGS += --build-arg=PKGS_PREFIX="$(PKGS_PREFIX)"
 # extra variables
 
 EXTENSIONS_IMAGE_REF ?= $(REGISTRY_AND_USERNAME)/extensions:$(TAG)
-PKGS ?= v1.9.0-alpha.0-4-ge90ae7e
+PKGS ?= v1.9.0-alpha.0-11-g232a153
```
Comment (smira) [inline Makefile:51]: this is generated, so we should update `.kres.yaml` and run `make rekres`

**Review by smira — COMMENTED** [review]
plus I think between the pkgs version there were other bumps, at least Linux firmware, which should be updated in `Pkgfile`

Also we update `hack/release.toml` now to say that previous is 1.8.0, and remove old notes, leaving just new component updates - that is, Linux firmware at the very least

**Comment by dsseng** [comment]
> plus I think between the pkgs version there were other bumps, at least Linux firmware, which should be updated in Pkgfile

None, the commit previously linked there was one where you updated the linux-firmware.

> Also we update hack/release.toml now to say that previous is 1.8.0, and remove old notes, leaving just new component updates - that is, Linux firmware at the very least

Should I do it in the same commit/PR?

**Comment by smira** [comment]
> None, the commit previously linked there was one where you updated the linux-firmware.

ok, good then!

**Review by smira — APPROVED** [review]

**Pull request merged**